TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -524,20 +417,33 @@ import PromiseKit
     }
 
     private func saveLinkedBrokerToKeychain(
-        userIdUserTokenBroker: UserIdUserTokenBroker,
+        linkedBrokerData: LinkedBrokerData,
         onSuccess: @escaping (_ linkedBroker: TradeItLinkedBroker) -> Void,
         onFailure: @escaping (TradeItErrorResult) -> Void
     ) {
         let linkedLogin = self.connector.saveToKeychain(
-            withUserId: userIdUserTokenBroker.userId,
-            andUserToken: userIdUserTokenBroker.userToken,
-            andBroker: userIdUserTokenBroker.broker,
-            andLabel: userIdUserTokenBroker.broker
+            withUserId: linkedBrokerData.userId,
+            andUserToken: linkedBrokerData.userToken,
+            andBroker: linkedBrokerData.broker,
+            andLabel: linkedBrokerData.broker
         )
 
         if let linkedLogin = linkedLogin {
             let linkedBroker = self.loadLinkedBrokerFromLinkedLogin(linkedLogin)
-            if userIdUserTokenBroker.isLinkActivationPending {
+            linkedBroker.accounts = linkedBrokerData.accounts.map { accountData in
+                return TradeItLinkedBrokerAccount(
+                    linkedBroker: linkedBroker,
+                    accountName: accountData.name,
+                    accountNumber: accountData.number,
+                    accountIndex: "",
```

Comment:
Yahoo should just ignore this property as far as I know, so default this to empty string here. Not sure if needs comment?